### PR TITLE
CleanCode

### PR DESCRIPTION
**Pull request by neringamaj**

### CODE REVIEW

File: ridalot2.0/ridalot2.0/Data/DBService.cs [hunk 1]
```diff
@@ -44,37 +45,37 @@ public DBService(RIDALOTContext context)
                  .AsNoTracking().ToListAsync();
         }
         public async Task<List<Posts>>
-            GetFeedPostsAsync(int strCurrentUser)
+            GetFeedPostsAsync(Status strCurrentUser)
```
Comment (Astijq) [inline ridalot2.0/ridalot2.0/Data/DBService.cs:48]: dont need the variable here

File: ridalot2.0/ridalot2.0/Data/Dimensions.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿namespace ridalot2._0.Data
```
Comment (Astijq) [inline ridalot2.0/ridalot2.0/Data/Dimensions.cs:1]: formating....

File: ridalot2.0/ridalot2.0/Data/FilterDimensions.cs [hunk 3]
```diff
@@ -0,0 +1,38 @@
+﻿namespace ridalot2._0.Data
```
Comment (Astijq) [inline ridalot2.0/ridalot2.0/Data/FilterDimensions.cs:1]: formating

File: ridalot2.0/ridalot2.0/Pages/CreateAnAd.razor [hunk 4]
```diff
@@ -163,25 +136,20 @@ else
...
 
-	public struct Dimensions
-	{
-		public int height { get; set; }
-		public int width { get; set; }
-		public int length { get; set; }
-		public int weight { get; set; }
-	}
-	Dimensions d;
+	///////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (Astijq) [inline ridalot2.0/ridalot2.0/Pages/CreateAnAd.razor:139]: ?

**Review by Astijq — APPROVED** [review]
branch doesnt match commits doesnt match pull request. but work is good

**Review by kambalas — APPROVED** [review]
ok!

**Pull request merged**